### PR TITLE
add starter pack to tutorial

### PR DESCRIPTION
**Pull request by akelad**
**Proposed changes**:
- add starter pack to the moodbot tutorial

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: docs/tutorial_basics.rst [hunk 1]
```diff
@@ -184,19 +184,19 @@ To train the dialogue model, run:
 
 .. code-block:: bash
 
-   python -m rasa_core.train -s data/stories.md -d domain.yml -o models/dialogue --epochs 300
```
Comment (tmbo) [inline docs/tutorial_basics.rst]: I don't think we should hide these.

File: docs/tutorial_basics.rst [hunk 2]
```diff
@@ -27,32 +27,33 @@ We will create a very simple bot that checks our current mood and tries to
 cheer us up if we are feeling sad. It will query our mood and based on our
 reply will respond with a funny image or a message.
 
-Let's start by creating a project folder:
+We created a `starter pack <https://github.com/RasaHQ/starter-pack>`_ to help
```
Comment (tmbo) [inline docs/tutorial_basics.rst]: So now we have that code twice? Once in this repo and once in the starter pack?

Comment (akelad) [reply docs/tutorial_basics.rst]: what do you mean? the starter pack is just the bare setup of files, the actual moodbot code is not present in there

File: docs/tutorial_basics.rst (hunk 1 above)
Comment (akelad) [reply docs/tutorial_basics.rst]: yeah fair point, i'll just paste the commands from the make file in here

File: docs/tutorial_basics.rst (hunk 2 above)
Comment (tmbo) [reply docs/tutorial_basics.rst]: ah ok, so you still need to create the data files? sounds good :+1: - was just worried we keep that data in two places.

Comment (akelad) [reply docs/tutorial_basics.rst]: no no, all good :D this is the starter pack from the live demo haha

**Pull request merged**